catalogue_pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/services/Merger.scala

```diff
@@ -54,7 +54,9 @@ class Merger extends Logging {
 
   private def mergePhysicalWithDigitalWork(physicalWork: UnidentifiedWork,
                                            digitalWork: UnidentifiedWork) = {
-    physicalWork.copy(items = physicalWork.items ++ digitalWork.items)
+    physicalWork.copy(items = List(physicalWork.items.head.copy(
```

Comment:
This would be easier to read with a few intermediate variables. How about:

```scala
val physicalItem = physicalWork.items.head
val digitalItem = digitalWork.items.head

val newItem = physicalItem.copy(
  locations = physicalItem.agent.locations ++ digitalItem.agent.locations
)

physicalWork.copy(items = List(newItem))
```